crypto/src/conversation.rs

```diff
@@ -302,9 +306,10 @@ impl MlsConversation {
     /// * `backend` - the KeyStore to persist possible group changes
     ///
     /// # Return type
-    /// This method will return None for the message in case the provided payload is
+    /// This method will return a tuple containing an optional message and an optional delay time
+    /// for the callers to wait for committing. A message will be `None` in case the provided payload is
     /// a system message, such as Proposals and Commits. Otherwise it will return the message as a
-    /// byte array
+    /// byte array. The delay will be `Some` only when the message contains a proposal
```

Comment:
I think the delay will also be Some in the case where we decrypt a commit and have pending proposals
```suggestion
    /// byte array. The delay will be `Some` when the message is a proposal. 
    TODO: It might also be Some in the case the message is a commit and we have local pending proposals not covered by this commit
```